src/zkevm_specs/evm_circuit/execution/error_gas_uint_overflow.py

```diff
@@ -32,7 +36,7 @@ def error_gas_uint_overflow(instruction: Instruction):
         is_log3,
         is_log4,
         is_sha3,
-        is_exp,
+        _,
```

Comment:
is_exp has dynamic gas of 50 * exponent_byte_size
I think we should put is_exp in the `is_dynamic_gas` variable below

Reply:
I found that it only takes care memory expansion cost which EXP opcode doesn't have. So, I just remove EXP in `ErrorGasUintOverflow` gadget. We might need to have more investigation on `ErrorGasUintOverflow` error state and we could do it when we apply Scroll's approch (performing `ErrorGasUintOverflow` check in each oog case)